app_dart/bin/gh.dart

```diff
@@ -0,0 +1,306 @@
+import 'dart:convert' show jsonEncode;
+
+import 'package:cocoon_service/src/datastore/cocoon_config.dart';
+import 'package:http/http.dart' as http;
+import 'package:graphql/client.dart';
+import 'package:github/server.dart';
+import 'package:meta/meta.dart';
+
+const String _removeLabelMutation = r'''
+mutation RemoveLabelAndComment($id: ID!, $sBody: String!, $labelId: ID!) {
+  addComment(input: { subjectId:$id, body: $sBody }) {
+    clientMutationId
+  }
+  removeLabelsFromLabelable(input: { labelableId: $id, labelIds: [$labelId] }) {
+    clientMutationId
+  }
+}''';
+
+Future<void> main() async {
+  final HttpLink _httpLink = HttpLink(
+    uri: 'https://api.github.com/graphql',
+  );
+
+  final AuthLink _authLink = AuthLink(
+    getToken: () async => 'Bearer fabddf154d0d1d80289845e98b2416cce96641b8',
+  );
+
+  final Link _link = _authLink.concat(_httpLink);
+
+  final GraphQLClient _client = GraphQLClient(
+    cache: InMemoryCache(),
+    link: _link,
+  );
+
+  const String _labeledPullRequestsWithReviewsQuery = r'''
```

Comment:
Is this part of the test? Why is it not using `labeledPullRequestsWithReviewsQuery` from check_for_waiting_pull_requests_queries.dart which I think should be the subject of the test?

Reply:
This was mistakenly checked in.  It's not used anywhere - was using it for testing queries against actual github.  You'll notice it even had an auto code from my account in there, which I have revoked  :)

Reply:
Hmm... then where's the test? Did you forget to add some files?

Reply:
I did.  It should be present now.